.github/workflows/ci.yml

```diff
@@ -388,6 +388,74 @@ jobs:
       run: cmake --build build --target test_cmake_build
 
 
+  # Testing on ICX/ICPX, Intel's next-gen C/C++ compiler using the oneAPI apt repo
+  icx:
```

Comment:
There is still something off here, I must have broken the yaml

Reply:
What happens if you `import yaml` and `yaml.load(open(".github/workflows/ci.yml"))`? That should catch syntax errors

Reply:
Nvm, just tested that, and that works :-(